probe-rs/Cargo.toml

```diff
@@ -64,6 +64,9 @@ svg = "0.13.1"
 thiserror = { workspace = true }
 tracing = { version = "0.1.37", features = ["log"] }
 
+espflash = { version = "2.0.0-rc.3", git = "https://github.com/MabezDev/espflash", branch = "new-esp-idf-part" }
```

Comment:
Does the `2.0.0-rc.3` version work, or only your branch?

If the crates.io version doesn't work, I'd prefer to have only the git reference here, so that we don't create a release with the wrong version by accident.